packages/vertica-nodejs/test/integration/client/error-handling-tests.js

```diff
@@ -18,24 +18,6 @@ var createErorrClient = function () {
 
 const suite = new helper.Suite('error handling')
 
-suite.test('sending non-array argument as values causes an error callback', (done) => {
```

Comment:
this is valid behavior now as we can pass a config in place of values for copy from local stdin